spring-security-mvc-boot/src/main/java/org/baeldung/rolesauthorities/persistence/PrivilegeRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.baeldung.rolesauthorities.persistence;
+
+import org.baeldung.rolesauthorities.model.Privilege;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface PrivilegeRepository extends JpaRepository<Privilege, Long> {
+
+    Privilege findByName(String name);
+
+    @Override

```

Comment:
Delete the @Override annotation, as it is atypical on an interface and does not do anything in this case. Same for the other repository interfaces.